app/controllers/discussions_controller.rb

```diff
@@ -13,7 +13,7 @@ def index
     end
 
   	@discussions = @discussions.search(params[:search]).order('created_at DESC')
-    @discussions = @discussions.tagged_with(params[:filter]) if params[:filter].present?
+    @discussions = @discussions.tagged_with(params[:filter], any: true) if params[:filter].present?
```

Comment:
Is there a way to filter and require that the discussions have all the tags (rather than just one of them)? Looked at the taggable gem and there's `:match_all => true` but it says it has to have that exact set of tags (will exclude any discussions with extra tags on top of that) - is there some sort of combination/join we can do with these taggable filters to basically "match_all" but allow discussions that have extra tags?